low-level/files/opt/seagate/sspl/setup/generate_sspl_bundle/sspl_bundle_generate

```diff
@@ -305,17 +299,16 @@ class SSPLBundleGenerate(object):
 
 if __name__ == "__main__":
     if not os.path.isfile(onGoingRequestPath):
-        if len(sys.argv) < 3:
-            print("Unrecognized arguments: %s" % sys.argv)
-            sys.exit(1)
-        tmpFile = open(onGoingRequestPath, "w")
-        os.makedirs(f"/tmp/{sspl_bundle_tmp}")
-        SSPLBundleObj = SSPLBundleGenerate()
-        SSPLBundleObj.generate_sspl_support_bundle(sys.argv, tmpFile)
-        tmpFile.close()
-        # Removed sspl support bundle temp folder.
-        shutil.rmtree(f"/tmp/{sspl_bundle_tmp}")
-        os.remove(onGoingRequestPath)
+        if len(sys.argv) >= 3:
+            tmpFile = open(onGoingRequestPath, "w")
+            os.makedirs(f"/tmp/{sspl_bundle_tmp}")
+            SSPLBundleObj = SSPLBundleGenerate()
+            SSPLBundleObj.generate_sspl_support_bundle(sys.argv, tmpFile)
+            tmpFile.close()
+            # Removed sspl support bundle temp folder.
+            shutil.rmtree(f"/tmp/{sspl_bundle_tmp}")
+            os.remove(onGoingRequestPath)
+        else:
+            print("ERROR: Unrecognized arguments: %s" % sys.argv)
```

Comment:
so this will not  fail the script, should have a non zero error code if error encountered